README.md

```diff
@@ -12,28 +12,62 @@ After you create a new repository using this repo as template, please follow the
 
     1.4 Follow the instructions from Github.
 
+
 2. Change the License if MIT license is not suitable for you project. For more information about licenses, please refer to [Choose an open source license](https://choosealicense.com/).
 
+
 3. Update the envrionment dependency file `environment.yml`, which contain both runtime and development dependencies. For more information about conda environment file, please refer to [Conda environment file](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-file-manually).
 
-    3.1 We strongly recommended using a single `environment.yml` file to manage all the dependencies, including the runtime and development dependencies.
-    3.2 Please add comments to the `environment.yml` file to explain the dependencies.
-    3.3 Please prune the dependencies to the minimum when possible, we would like the solver to figure out the dependency tree for us.
+    3.1 Specify environment 'name' field to match package name
+
+    3.2 We strongly recommended using a single `environment.yml` file to manage all the dependencies, including the runtime and development dependencies.
+
+    3.3 Please add comments to the `environment.yml` file to explain the dependencies.
+
+    3.4 Please prune the dependencies to the minimum when possible, we would like the solver to figure out the dependency tree for us.
 
 
 4. Adjust pre-commit configuration file, `.pre-commit-config.yaml` to enable/disable the hooks you need. For more information about pre-commit, please refer to [pre-commit](https://pre-commit.com/).
 
+
 5. Having code coverage, `codecov.yaml` is **strongly recommended**, please refer to [Code coverage](https://coverage.readthedocs.io/en/coverage-5.5/) for more information.
 
+
 6. Adjust the demo Github action yaml files for CI/CD. For more information about Github action, please refer to [Github action](https://docs.github.com/en/actions).
 
+    6.1 Specify package name at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/.github/workflows/package.yml#L34
+
+    6.2 Specify package name at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/.github/workflows/package.yml#L46
```

Comment:
Instead of using the link to a particular SHA, how about just the file path, something like: `PROJECT_ROOT/.github/workflows/package.yml#L34`?